src/Adapter/Product/QueryHandler/GetProductForEditingHandler.php

```diff
@@ -107,6 +115,7 @@ public function __construct(
         ProductRepository $productRepository,
         StockAvailableRepository $stockAvailableRepository,
         VirtualProductFileRepository $virtualProductFileRepository,
+        AttachmentRepository $attachmentRepository,
```

Comment:
Please add it at the end as it a new parameter.

And it's a BC Break

Reply:
It's part of the experimental page, so the place of the argument changes nothing It makes sense to keep all the repositories first